merlin/systems/dag/ops/fil.py

```diff
@@ -384,6 +384,9 @@ def get_fil_model(model) -> FILModel:
         ),
     ):
         fil_model = SKLearnRandomForest(model)
+    elif hasattr(model, "booster"):
```

Comment:
We should move this up with the other xgboost models in the elif.

Also not entirely clear why:

>     elif xgboost and isinstance(model, xgboost.XGBModel):
>         fil_model = XGBoost(model.get_booster())

isn't picking up this case.  Am I correct in understanding that they're both trying to get the booster from the model?  Or is this a different case.

Reply:
`isinstance(model, xgboost.XGBModel)` doesn't pick up the merlin models case because it doesn't inherit from `xgboost.XGBModel`.

Reply:
> lightgbm merlin model case

We don't have LightGBM integration in Merlin Models currently. It would be a similar implementation to the XGBoost case though.

Reply:
✅  moved this line up to be alongside the other xgboost models in the elif as suggested